--- Config.h	2016-11-22 08:29:27.332787245 -0800
+++ Config.h.cmssw.endcap	2016-11-22 08:37:33.009820187 -0800
@@ -33,12 +33,12 @@
   extern int nEvents;
 
   // config on main -- for geometry
-  constexpr int   nLayers   = 10; // default: 10; cmssw tests: 13, 17, 26 (for endcap)
+  constexpr int   nLayers   = 26; // default: 10; cmssw tests: 13, 17, 26 (for endcap)
   constexpr float fRadialSpacing   = 4.;
   constexpr float fRadialExtent    = 0.01;
   constexpr float fInnerSensorSize = 5.0; // approximate sensor size in cm
   constexpr float fOuterSensorSize = Config::fInnerSensorSize * 2.;
-  constexpr float fEtaDet          = 1; // default: 1; cmssw tests: 2, 2.5
+  constexpr float fEtaDet          = 2.5; // default: 1; cmssw tests: 2, 2.5
 
   //constexpr float cmsAvgRads[13] = {4.42,7.31,10.17,25.58,33.98,41.79,49.78,60.78,69.2,77.96,86.80,96.53,108.00}; // cms average radii, noSplit version
   constexpr float cmsAvgRads[17] = {4.42,7.31,10.17,25.58,25.58,33.98,33.98,41.79,49.78,60.57,61.00,69.41,68.98,77.96,86.80,96.53,108.00}; // cms average radii, split version
@@ -106,9 +106,9 @@
   static constexpr float m_max_dphi = 0.02; // default: 0.02; cmssw tests: 0.2
 
   // config on Event
-  constexpr float chi2Cut = 15.;// default: 15.; cmssw tests: 30.
+  constexpr float chi2Cut = 30.;// default: 15.; cmssw tests: 30.
   constexpr float nSigma  = 3.;
-  constexpr float minDPhi = 0.;// default: 0.;  cmssw tests: 0.01;
+  constexpr float minDPhi = 0.01;// default: 0.;  cmssw tests: 0.01;
   constexpr float maxDPhi = Config::PI;
   constexpr float minDEta = 0.;
   constexpr float maxDEta = 1.0;
